src/main/webapp/app/exercises/shared/statistics/doughnut-chart.component.ts

```diff
@@ -101,4 +85,26 @@ export class DoughnutChartComponent implements OnChanges, OnInit {
             this.router.navigate(this.titleLink);
         }
     }
+
+    /**
+     * Asigns a given array of numbers to ngxData
+     * @param values the values that should be displayed by the chart
+     * @private
+     */
+    private assignValuesToData(values: number[]) {
+        this.ngxDoughnutData[0].value = values[0];
+        this.ngxDoughnutData[1].value = values[1];
+        this.ngxDoughnutData = [...this.ngxDoughnutData];
+    }
+
+    /**
+     * Modifies the tooltip content of the chart.
+     * Returns absolute value represented by doughnut piece or 0 if the currentMax is 0.
+     * This is necessary in order to compensate the workaround for
+     * displaying a chart even if no values are there to display (i.e. currentMax is 0)
+     * @param value the default tooltip content that has to be replaced
+     */
+    valueFormatting(value: any) {
```

Comment:
return is missing in the signature and the doc

Reply:
Done in edf3ef8